### PR TITLE
Better error handling / cli colors / avg compile time

### PR DESCRIPTION
**Pull request by marcauberer**
- Print compile errors in a more controlled and user-friendly way
- Do not redirect compile errors into output files
- Add average compile time output in addition to the total compile time for benchmark mode
- Add color to the cli output: red for errors, yellow for info, green for success, white for arbitrary

### CODE REVIEW

File: cli/preprocessor.go [hunk 1]
```diff
@@ -100,9 +98,9 @@ func analyze(
 	} else {
 		// Ensure value of comment char
 		if *lineCommentIden == "" && *blockCommentIdenOpen == "" && *blockCommentIdenClose == "" {
-			log.Fatal("You must provide at least one of line comment or block comments identifiers.")
+			util.Error("You must provide at least one of line comment or block comments identifiers.", true)
```
Comment (Splines) [inline cli/preprocessor.go:101]: what's the `true` argument for?

**Review by Splines — COMMENTED** [review]
Colors 😍

File: cli/preprocessor.go (hunk 1 above)
Comment (marcauberer) [reply cli/preprocessor.go:101]: Look here:
https://github.com/compose-generator/ccom/pull/44/files#diff-15a25de69b863b267217c2db6ca7b09a36fe3dc0c5f36a02b031bf16c5681124R32-R34

**Comment by marcauberer** [comment]
@Splines can you approve?

**Review by Splines — APPROVED** [review]

**Pull request merged**